lib/telephony/alert_sender.rb

```diff
@@ -46,6 +46,14 @@ def send_personal_key_sign_in_notice(to:, country_code:)
       response
     end
 
+    def send_raw_message(to:, message:, country_code:)
+      response = adapter.send(message: message, to: to, country_code: country_code)
+      log_response(response, context: __method__.to_s.gsub(/^send_/, ''))
+      response
+    end
+
+    alias send_notification send_raw_message
```

Comment:
```suggestion
    alias_method :send_notification, :send_raw_message
```

I think that `alias_method` is generally preferred because the scoping makes more sense: https://www.bigbinary.com/blog/alias-vs-alias-method

Reply:
@zachmargolis , changed.